projects/driver_controls/src/main.c

```diff
@@ -31,16 +87,48 @@ int main() {
 
   i2c_init(I2C_PORT_1, &settings);
 
-  GPIOAddress int_pin = { GPIO_PORT_A, 9 };
-  gpio_expander_init(&s_expander, I2C_PORT_1, GPIO_EXPANDER_ADDRESS_1, &int_pin);
+  GPIOAddress stalk_int_pin = { GPIO_PORT_A, 2 };
+  GPIOAddress stalk_ready_pin = { GPIO_PORT_A, 1 };
+  gpio_expander_init(&s_stalk_expander, I2C_PORT_1, GPIO_EXPANDER_ADDRESS_0, &stalk_int_pin);
+  ads1015_init(&s_stalk_ads1015, I2C_PORT_1, ADS1015_ADDRESS_VDD, &stalk_ready_pin);
+  control_stalk_init(&s_stalk, &s_stalk_ads1015, &s_stalk_expander);
+
+  GPIOAddress console_int_pin = { GPIO_PORT_A, 9 };
+  gpio_expander_init(&s_console_expander, I2C_PORT_1, GPIO_EXPANDER_ADDRESS_1, &console_int_pin);
+  center_console_init(&s_console, &s_console_expander);
+
+  GPIOAddress pedal_ads1015_ready = { GPIO_PORT_A, 10 };
+  ads1015_init(&s_pedal_ads1015, I2C_PORT_1, ADS1015_ADDRESS_GND, &pedal_ads1015_ready);
+  throttle_init(throttle_global(), &calib_blob(&s_calib)->throttle_calib, &s_pedal_ads1015);
+
+  cruise_init(cruise_global());
+  drive_output_init(drive_output_global(), INPUT_EVENT_DRIVE_WATCHDOG_FAULT,
+                    INPUT_EVENT_DRIVE_UPDATE_REQUESTED);
 
-  center_console_init(&s_console, &s_expander);
+  // TODO(ELEC-455): Add heartbeat handlers
```

Comment:
Please add it as it is one line for the powertrain one at least?

Reply:
I'll add BPS fault in the signal PR